de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/SarosSessionManager.java

```diff
@@ -742,18 +742,19 @@ private boolean terminateNegotiations() {
             negotiation.localCancel(null, CancelOption.NOTIFY_PEER);
         }
 
-        for (ProjectNegotiation negotiation : currentProjectNegotiations.list())
+        for (ReferencePointNegotiation negotiation : currentReferencePointNegotiations
+            .list())
             negotiation.localCancel(null, CancelOption.NOTIFY_PEER);
 
-        log.trace("waiting for all session and project negotiations to terminate");
+        log.trace("waiting for all session and referencePoint negotiations to terminate");
 
         long startTime = System.currentTimeMillis();
 
         boolean terminated = false;
 
         while (System.currentTimeMillis() - startTime < NEGOTIATION_TIMEOUT) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Reduce the total number of break and continue statements in this loop to use at most one. ([squid:S135](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWcmJqzpkEeD8QRkTSt0&open=AWcmJqzpkEeD8QRkTSt0)

[ISSUE_KEY]: AWcmJqzpkEeD8QRkTSt0